lte/gateway/c/core/oai/test/spgw_task/spgw_test_util.cpp

```diff
@@ -14,23 +14,55 @@
 #include "spgw_test_util.h"
 
 extern "C" {
-#include "itti_free_defined_msg.h"
+#include "lte/gateway/c/core/oai/lib/3gpp/3gpp_23.003.h"
+#include "lte/gateway/c/core/oai/lib/3gpp/3gpp_24.007.h"
+#include "lte/gateway/c/core/oai/lib/3gpp/3gpp_29.274.h"
+#include "lte/gateway/c/core/oai/tasks/nas/api/mme/mme_api.h"
+#include "lte/gateway/c/core/oai/include/s11_messages_types.h"
+#include "lte/gateway/c/core/oai/common/itti_free_defined_msg.h"
 }
 
 namespace magma {
 namespace lte {
 
 extern task_zmq_ctx_t task_zmq_ctx_main_spgw;
 
+bool expect_num_sessions(
```

Comment:
```suggestion
bool is_num_sessions_valid(
```
minor nit

Reply:
@ardzoht changed the function name in latest commit. Also expanded file paths for all includes and fixed the type for imsi64